arches/app/utils/permission_backend.py

```diff
@@ -293,7 +310,10 @@ def check_resource_instance_permissions(user, resourceid, permission):
     try:
         resource = ResourceInstance.objects.get(resourceinstanceid=resourceid)
         result["resource"] = resource
-        all_perms = get_perms(user, resource)
+
+        permissions_checker = CachedObjectPermissionChecker(user, ResourceInstance)
+        all_perms = permissions_checker.get_perms(resource)
```

Comment:
I've found that this can be stale in some cases allowing access to a restricted instance. To reproduce, create a new instance with admin by adding a single tile. Then 'Manage Permissions' > 'Configure Access to this Instance'. Next paste the new instance's url into another browser in which a different user is logged in and notice that the resource editor loads the instance. It should return a 403 instead. 

Maybe the permissions cache should be cleared when resource instance permissions are updated, or maybe it's worth considering not using the cache here?

Reply:
I think it would be fine to bypass the cache in the case of resource instance permissions for now, and try to address resource instance permission caching as part of #8021

Reply:
added a comment in #8021